docker/docker-compose.yml

```diff
@@ -35,9 +35,9 @@ services:
     image: fidals/se-nodejs:dev
     volumes:
       # Volumes for refarm-site's front development
-      - ../gulpfile.babel.js:/usr/app/src/gulpfile.babel.js
+#      - ../gulpfile.babel.js:/usr/app/src/gulpfile.babel.js
```

Comment:
Let's undo changes from this file, seems they will broke `collectstatic`

Reply:
sorry, forgot it